compiler/rustc_expand/messages.ftl

```diff
@@ -10,6 +10,11 @@ expand_attribute_meta_item =
 expand_attribute_single_word =
     attribute must only be a single word
 
+expand_attributes_on_expressions_experimental =
+    attributes on expressions are experimental
+    .help_outer_doc = `///` is for documentation comments. For a plain comment, use `//`.
+    .help_inner_doc = `//!` is for documentation comments. For a plain comment, insert a space: `// !`
```

Comment:
can you make this say "use // or insert a space"? i think it's likely that the ! could be erroneous too